into/tests/test_pydot.py

```diff
@@ -6,4 +7,9 @@ def test_create_pydot_image():
         dot_graph()
     except:
         pass
-
+    finally:
+        for ext in ('pdf', 'png'):
+            try:
+                os.remove('conversions.%s' % ext)
+            except:
+                pass
```

Comment:
The `into/dot.py` file is itself a script.  We can remove the auto-generation of graphs.  I can put this into my personal workflow and stop polluting everyone elses.